docs/snippets/ov_auto.py

```diff
@@ -8,7 +8,6 @@
 import openvino.properties.device as device
 import openvino.properties.hint as hints
 import openvino.properties.streams as streams
-import properties.enable_profiling as enable_profiling
```

Comment:
maybe we can correct it as below code.
`import openvino.properties.enable_profiling as enable_profiling`